### PR TITLE
added create item endpoint

### PR DESCRIPTION
**Pull request by Kirei-Nan**
Made the following changes:
-Added create item endpoint in routes.py
-Added test create item endpoint in test_routes.py

### CODE REVIEW

**Review by Hyxpillow — APPROVED** [review]
Looks good to me.

**Review by anuva98 — COMMENTED** [review]
LGTM. I am also including test_order.py and test_item.py in my upcoming PR. Can we all please update that as well?

**Review by anuva98 — APPROVED** [review]
For now we can merge this. I was going to create a PR with some duplication with this PR but I'll remove that then.

File: tests/test_models.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def test_example_replace_this(self):
         self.assertIsNotNone(resource.id)
         found = Order.all()
         self.assertEqual(len(found), 1)
-        data = Order.find(resource.id)
-        self.assertEqual(data.name, resource.name)
+        # data = Order.find(resource.id)
+        # self.assertEqual(data.name, resource.name)
```
Comment (anuva98) [inline tests/test_models.py:77]: Maybe we can remove this test case altogether?

**Review by anuva98 — APPROVED** [review]

File: tests/test_models.py (hunk 1 above)
Comment (Kirei-Nan) [reply tests/test_models.py:77]: sure

**Pull request merged**